### PR TITLE
Implement Navigation Bar

### PR DESCRIPTION
**Pull request by Ovifer13**
**Issue #48**

- [x] Create `Navbar` component
- [x] Create `layout` component
- [x] Import `layout` component to `_app.tsx`
- [x] Polish Navbar
- [x] Add Logo

### CODE REVIEW

**Comment by Ovifer13** [comment]
**Implementation proof**
[Navigation Bar setlife-solutions - Watch Video](https://www.loom.com/share/24c297df583c4b2c87a3215a98578af1)

File: components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                <a className="text-2xl font-bold my-4 text-primary" href="/conssultation" onClick={() => setTimeout(() => {setOpen(!open)}, 100)}>
+                    Consultation
+                </a>
+                <a className="text-2xl font-bold my-4 text-primary" href="/portfolio" onClick={() => setTimeout(() => {setOpen(!open)}, 100)}>
+                    Portfolio
+                </a>
+                <a className="text-2xl font-bold my-4 text-primary" href="/service-packages" onClick={() => setTimeout(() => {setOpen(!open)}, 100)}>
+                    Service package
+                </a>
+            </div>
```
Comment (sofiaromorales) [inline components/Navbar.tsx:44]: Convert this into a map

File: components/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                <div 
+                    className="z-50 flex relative w-8 h-8 flex-col justify-between items-center" 
+                    onClick={() => {
+                        setOpen(!open)
+                    }}
+                >
+                    {/* hamburger button */}
+                    <span className={`h-0.5 w-full bg-solid-white rounded-lg transform transition duration-300 ease-in-out ${open ? "rotate-45 translate-y-3.5" : ""}`} />
+                    <span className={`h-0.5 w-full bg-solid-white rounded-lg transition-all duration-300 ease-in-out ${open ? "w-0" : "w-full"}`} />
+                    <span className={`h-0.5 w-full bg-solid-white rounded-lg transform transition duration-300 ease-in-out ${open ? "-rotate-45 -translate-y-3.5" : ""}`} />
```
Comment (sofiaromorales) [inline components/Navbar.tsx:76]: Transform into its own component

**Review by sofiaromorales — CHANGES_REQUESTED** [review]

**Comment by sofiaromorales** [comment]
@Ovifer13 pls fix listing issues and take a look into the changes I made on the last commit.

Besides of that pls address the following changes:

1. Fix spacing on hamburger menu 
Screen Shot 2022-08-05 at 12 17 54 AM
2. On desktop the navbar has a different style
Screen Shot 2022-08-05 at 12 18 53 AM

**Comment by Ovifer13** [comment]
@sofiaromorales I see, Idk why I can't find where the desktop version is. I only have the mobile version and I thought It was going to be used for both.

**Comment by sofiaromorales** [comment]
> @sofiaromorales I see, Idk why I can't find where the desktop version is. I only have the mobile version and I thought It was going to be used for both.

https://www.figma.com/file/KlLr27PTO7Dzz3D9CE2LzT/setlife-core?node-id=649%3A1467

On the same figma file to the left you have a panel with multiple pages

**Comment by sofiaromorales** [comment]
@Ovifer13 pls fix linter bugs and have in consideration the changes I made in [c8a380d](https://github.com/setlife-network/setlife-solutions/pull/103/commits/c8a380d4be63ce320991b249a3b2270de5223aec) for future pr's

**Comment by Ovifer13** [comment]
**Implementation proof with fixes**
[Implement Navigation Bar after fixes - Watch Video](https://www.loom.com/share/48c16c748f62497bbc9c57b714c111e8)

**Review by sofiaromorales — APPROVED** [review]

**Pull request merged**